src/util-runmodes.c

```diff
@@ -49,6 +49,14 @@
 
 #include "flow-hash.h"
 
+#ifdef HAVE_DPDK
+#include <rte_lcore.h>
+#include <rte_ethdev.h>
+#endif /* HAVE_DPDK */
+
+// equal to -1 to get the first lcore id
+int last_spawned_lcore = -1;
```

Comment:
Should be in the ifdef ?

Reply:
True, it is irrelevant to all runmodes other than the DPDK runmode. I will move it there, thanks!